src/ui/components/Passport/Passport.tsx

```diff
@@ -231,20 +232,20 @@ const Passport = (props: PropsFromRedux) => {
 
   const sections: Section[] = [
     {
-      title: "TIME TRAVEL",
+      title: "TIME TRAVELLER",
       items: timeTravelItems,
     },
     {
-      title: "INFRARED INSPECTION",
-      items: inspectionItems,
+      title: "JUMP EXPERT",
+      items: jumpItems,
     },
     {
-      title: "SWISS ARMY KNIFE",
-      items: swissArmyItems,
+      title: "INSPECTOR",
+      items: inspectionItems,
     },
     {
-      title: "MULTIPLAYER",
-      items: multiplayerItems,
+      title: "ADVENTURER",
+      items: swissArmyItems,
```

Comment:
I am using the legacy name here for now because I'm pretty sure we're going to end up changing the user-facing copy 3 more times.